### DISCUSSION TITLE
(CSS styling) Move the TOC (right sidebar) more to the right to make more space for the article

### DISCUSSION THREAD

**Discussion by ZenaMel** [facebook/docusaurus#8799] [Q&A] [answered]
Loosely related to #8794 

I want to move the TOC on the right sidebar more to the right so I can have my main content take up more space in the center without it overlapping with the TOC.

I found the `theme-doc-toc-desktop` CSS accessor via [the reference](https://docusaurus.io/docs/styling-layout#theme-class-names) but I'm not sure what attributes to change.
Looks like the TOC is nested under a col class that actually handles the padding/margins of the TOC sidebar. Maybe there's an accessor similar to `docSidebarContainer` but for the right sidebar container?

![image](https://user-images.githubusercontent.com/47085752/226551550-0cecac6d-dcea-421f-903f-b386c09685bf.png)

**Comment by ZenaMel** [comment]
**What I tried so far:**
Add margin-left (130px) to .theme-doc-toc-desktop

**Result:**
![image](https://user-images.githubusercontent.com/47085752/226553558-bcd728f7-e880-4fce-a715-86f67d9b8fbb.png)

I haven't found a way to remove the space on the right and it seems like this isn't possible by styling the theme-doc-toc-desktop class.

**Comment by slorber** [comment, marked as answer]
This component should be swizzled, and give you full power to modify


```js
export default function DocItemLayout({children}: Props): JSX.Element {
  const docTOC = useDocTOC();
  const {
    metadata: {unlisted},
  } = useDoc();
  return (
    <div className="row">
      <div className={clsx('col', !docTOC.hidden && styles.docItemCol)}>
        {unlisted && <Unlisted />}
        <DocVersionBanner />
        <div className={styles.docItemContainer}>
          <article>
            <DocBreadcrumbs />
            <DocVersionBadge />
            {docTOC.mobile}
            <DocItemContent>{children}</DocItemContent>
            <DocItemFooter />
          </article>
          <DocItemPaginator />
        </div>
      </div>
      {docTOC.desktop && <div className="col col--3">{docTOC.desktop}</div>}
    </div>
  );
}
```

At the end of the day, it's your responsibility to figure out the CSS to achieve your custom layout. We only provide a default component and CSS.

If we can refactor this component further to help you achieve your need in a more maintainable way, your feedback is welcome.
